docs/tutorials/manifest-parser.md

```diff
@@ -346,8 +346,8 @@ MyManifestParser.prototype.loadReference_ =
 
 If your content is encrypted, there are a few changes to the manifest you need
 to do.  First, for each Stream that contains encrypted content, you need to set
-`stream.encrypted` to true and put the key IDs that the stream is encrypted with
-in the `stream.keyIds` set.  Filling out the `keyIds` is technically optional,
+`stream.encrypted` to true and put the key IDs that the stream is encrypted within
```

Comment:
No, "with in" was correct. You are taking the "key IDs that the stream is encrypted with" and putting them "in the `stream.keyIds` set".

Reply:
Makes sense. Will update.